vortexasdk/api/voyages.py

```diff
@@ -144,21 +160,27 @@ class VoyageEnrichedItem(FromDictMixin):
 
     """
 
-    schema_version: str
+    schema_version: Optional[str] = None
     voyage_id: ID
-    start_timestamp: Optional[ISODate]
-    end_timestamp: Optional[ISODate]
-    start_event_id: ID
-    end_event_id: Optional[ID]
+    start_event_id: Optional[ID] = None
     vessel_id: ID
-    previous_voyage_id: Optional[ID]
-    next_voyage_id: Optional[ID]
-    latest_product_ids: List[ID]
-    tags: List[Tag]
-    tonne_miles: Optional[int]
-    vessel: VoyagesVesselEntity
-    corporate_entities: List[Entity]
-    odometer_start: Optional[int]
-    odometer_end: Optional[int]
-    events: List[Optional[Union[VoyageStatusEvent, VoyageVesselEvent, VoyageCargoEvent]]]
-    latest_product_details: List[Entity]
+    vessel: Optional[VoyagesVesselEntity] = None
+    start_timestamp: Optional[ISODate] = None
+    end_timestamp: Optional[ISODate] = None
+    end_event_id: Optional[ID] = None
+    previous_voyage_id: Optional[ID] = None
+    next_voyage_id: Optional[ID] = None
+    latest_product_ids: Optional[List[ID]] = Field(default_factory=list)
+    tags: Optional[List[Tag]] = Field(default_factory=list)
+    tonne_miles: Optional[int] = None
+    corporate_entities:Optional[List[EntityWithSingleLayerAndTimespan]] = Field(
+        default_factory=list
+    )
+    odometer_start: Optional[int] = None
+    odometer_end: Optional[int] = None
+    events: Optional[List[
+        Optional[Union[VoyageVesselEvent, VoyageCargoEvent, VoyageStatusEvent]]
```

Comment:
Can you please add a comment about the fact that the order of the classes matters?

Reply:
Done.